0000-server-roles.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Interledger server roles
+draft: 1
+---
+# Interledger server roles
+
+This document describes the different roles a server can play on the Interledger main net or testnet: connector, shop, and payee.
```

Comment:
A shop is a payee. I think those role names could be better. It's also not clear from the descriptions what the differences are. A real world example or use case would help

Reply:
Agreed! I'll get rid of the distinction between a payee (human-negotiated payment setup, so PaymentPointer/WebFinger/SPSP-URL in email, phone conversation, etc.) and a shop (machine-negotiated payment setup where humans only click, so http-ilp/web-monetization/web-payments). The distinction of "whether the payment setup data passes through human hands" is a bit arbitrary, for instance copy-paste of an opaque blob or scanning a QR code is somewhere inbetween clicking a shop's Pay button and typing a payee's address.